rllib/utils/actor_manager.py

```diff
@@ -228,12 +228,17 @@ def add_actors(self, actors: List[ActorHandle]):
             self.__NEXT_ID += 1
 
     @DeveloperAPI
-    def remove_actor(self, actor_id: int):
+    def remove_actor(self, actor_id: int) -> ActorHandle:
```

Comment:
actor_id or index?

Actually, dumb question: How to we handle failed workers now "in the middle"? Like num_rollout_workers=10, worker 5 failed, we create a new one with what index/ID? 5? or 11?

Reply:
at actor_manager level, everything is indexed by ID.
at worker_set level though, for backward compatibility reasons, I am still using indices as the names.
don't want to potentially break some code somewhere.

good question about recovery. in the new world, we don't actually re-create workers ourselves, Core does it.
so ID and actor handle doesn't have to be updated at all.

Reply:
so you'll reuse the failed actor id? How does that work? Do you not increment the id until the actor is added?